Fusion2/Fusion2.py

```diff
@@ -4,7 +4,7 @@
 import vtk, qt, ctk, slicer
 from slicer.ScriptedLoadableModule import *
 from slicer.util import VTKObservationMixin
-import nrrd
```

Comment:
Is this already in #53?

Reply:
Yes, I did not change anything in Fusion2 (or any unrelated files) for this pull request. Only .gitignore, .readthedocs.yaml and docs/

Reply:
I have removed the irrelevant commit from this pull request.